html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head><title>html forms</title></head>
+  <body>
+    <h1>  Coding Form! </h1>
+    <div>
+      <form action="http://email.com/email/php">
+
+        <label>Email: </label>
+        <div>
+          <input type="text" name="Email" size="12" maxlength="25"/>
+        </div>
+        <label>Why is coding cool </label>
+        <div>
+          <select name="Option">
+            <option value="interesting">Because it's interesting</option>
+            <option value="radio">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select> </div>
+
+        <p> <label> Favorite Language (choose one) </label> </p>
+        <div>
+          <label>
+            <input type="radio" name="genre" value="HTML" /> HTML </label>
+          <label>
+            <input type="radio" name="genre" value="CSS" />
+            CSS </label>
+          <label>
+            <input type="radio" name="genre" value="Javascript" />
+            Javascript </label>
+        </div>
```

Comment:
Your indentation got a little crazy here, make sure you stay consistent so you don't accidentally end up with bugs 😬